pkg/storage/pebble.go

```diff
@@ -2506,15 +2506,14 @@ func (p *Pebble) CreateCheckpoint(dir string, spans []roachpb.Span) error {
 // named version, it can be assumed all *nodes* have ratcheted to the pebble
 // version associated with it, since they did so during the fence version.
 var pebbleFormatVersionMap = map[clusterversion.Key]pebble.FormatMajorVersion{
-	clusterversion.V25_4_PebbleFormatV2BlobFiles: pebble.FormatV2BlobFiles,
-	clusterversion.V25_3:                         pebble.FormatValueSeparation,
+	clusterversion.V25_4: pebble.FormatV2BlobFiles,
```

Comment:
`V25_4_PebbleFormatV2BlobFiles` has been changed to `V25_4`, as discussed in https://github.com/cockroachdb/cockroach/pull/158225#discussion_r2557425028